schedule/yast/maintenance/yast_mru_install_minimal_with_addons_aarch64.yaml

```diff
@@ -0,0 +1,11 @@
+---
```

Comment:
`*_aarch64_dev.yaml` we should keep that naming before deploying, it would help to distinguish them.